src/webapp/src/client/resources/lt_LT/settings.json

```diff
@@ -22,5 +22,6 @@
 	"comments": "Komentarai",
 	"myPosts": "Mano įrašai",
 	"iCommented": "Sekami įrašai",
+	"mention": "Kai esu paminimas",
```

Comment:
"Kai esu paminėtas" perhaps? Or to keep better consistency with other notification settings - "Mano vardo paminėjimai"?